scripts/install_jetson.sh

```diff
@@ -22,8 +22,8 @@ fi
 
 # Set up CUDA environment
 if [ ! -x "$(command -v nvcc)" ]; then
-    echo "export PATH=/usr/local/cuda/bin${PATH:+:${PATH}}" >> ~/.bashrc
-    echo "export LD_LIBRARY_PATH=/usr/local/cuda/lib64${LD_LIBRARY_PATH:+:${LD_LIBRARY_PATH}}" >> ~/.bashrc
+    echo "export PATH=/usr/local/cuda/bin\${PATH:+:\${PATH}}" >> ~/.bashrc
+    echo "export LD_LIBRARY_PATH=/usr/local/cuda/lib64\${LD_LIBRARY_PATH:+:\${LD_LIBRARY_PATH}}" >> ~/.bashrc
```

Comment:
Why adding the backslashes?

Reply:
Executing the script resolves the variable values before exporting it to the _.basrc_ file. This means the _.bashrc_ file contains and exports the path values like they are at the moment the script was executed forever.
To just add the cuda path dynamically the variables should be resolved again every time _.bashrc_ gets called. Therefore the variable definition needs to be escaped.

Reply:
It adds the current path variables to the bashrc, which will set them forever like they are at this moment.